sympy/core/operations.py

```diff
@@ -604,7 +605,7 @@ def dispatch(self, *types):
         # Sieve the types with low op_priority or no handler
         # This makes it faster than using dispatcher on every types.
         sieved_data = tied_max(
-            (typ for typ in types if getattr(typ, self.handlermethod, None) is not None),
+            (typ for typ in types if hasattr(typ, self.handlermethod)),
```

Comment:
This change makes it impossible for a subclass to disable the handler of its base class.

Reply:
I overlooked that.